doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -82,175 +76,107 @@ To work with the nRF54H20 DK, follow the instructions in the next sections to in
-
-1. In the terminal window opened during the toolchain installation, initialize west with the revision of the |NCS| from the customer sampling:
+Installing the Terminal application
+***********************************
 
-   .. parsed-literal::
-      :class: highlight
+On your computer, install `nRF Connect for Desktop`_.
+You must also install a terminal emulator, such as `nRF Connect Serial Terminal`_ (from the nRF Connect for Desktop application) or the nRF Terminal (part of the `nRF Connect for Visual Studio Code`_ extension).
 
-      west init -m https://github.com/nrfconnect/sdk-nrf --mr v2.6.99-cs2
+Installing nRF Util and its commands
+************************************
 
-#. Enter the following command to clone the project repositories::
+Using the nRF54H20 DK with the |NCS| v2.7.0 requires the following:
 
-      west update
+* nRF Util version 7.11.1 or above
+* nRF Util ``device`` version 2.4.0
 
-   Depending on your connection, this might take some time.
+1. Download the nrfutil executable file from the `nRF Util development tool`_ product page.
+#. Add nRF Util to the system path (on Linux and MacOS) or environment variables (on Windows) to run it from anywhere on the system.
+   On Linux and MacOS, use one of the following options:
 
-#. Export a :ref:`Zephyr CMake package <zephyr:cmake_pkg>`.
-   This allows CMake to automatically load the boilerplate code required for building |NCS| applications::
+   * Add nRF Util's directory to the system path.
+   * Move the file to a directory in the system path.
 
-      west zephyr-export
+#. On MacOS and Linux, give ``nrfutil`` execute permissions by typing ``chmod +x nrfutil`` in a terminal or using a file browser.
+   This is typically a checkbox found under file properties.
+#. On MacOS, To run the nrfutil executable you need to allow it in the system settings.
+#. Verify the version of the nRF Util installation on your machine by running the following command::
 
-Your directory structure now looks similar to this::
+      nrfutil --version
 
-    ncs-lcs/work-dir
-    |___ .west
-    |___ bootloader
-    |___ modules
-    |___ nrf
-    |___ nrfxlib
-    |___ zephyr
-    |___ ...
+#. If your version is below 7.11.1, run the following command to update nRF Util::
 
-Note that there are additional folders, and that the structure might change.
-The full set of repositories and folders is defined in the manifest file.
+      nrfutil self-upgrade
 
-Installing the Terminal application
-***********************************
+#. Install the nRF Util ``device`` command to version 2.4.0 as follows::
 
-On your computer, install `nRF Connect for Desktop`_.
-You must also install a terminal emulator, such as `nRF Connect Serial Terminal`_ (from the nRF Connect for Desktop application) or the nRF Terminal (part of the `nRF Connect for Visual Studio Code`_ extension).
+      nrfutil install device=2.4.0 --force
```

Comment:
In addition, link here: https://nordic-dev.zoominsoftware.io/bundle/nrfutil/page/guides/installing_commands.html

I'll make sure installing with ``--force`` is documented there before v2.7.0 is live.

Reply:
If you want to ensure that exactly version 2.4.0 is installed, then `--force` is needed. If `--force` is not specified, then this command will only install nrfutil-device if it is not yet installed, and not do anything if any version of nrfutil-device is already installed.